test/xUnit/csharp/test_Subsystem.cs

```diff
@@ -86,8 +85,7 @@ public static void RegisterSubsystem()
 
                 // Now validate the all-subsystem-info collection.
                 ReadOnlyCollection<SubsystemInfo> ssInfos = SubsystemManager.GetAllSubsystemInfo();
-                Assert.Single(ssInfos);
-                Assert.Same(ssInfos[0], ssInfo);
```

Comment:
Please keep this test.

Reply:
I guess you don't need to add more tests for the `ICrossPlatformDsc` in `RegisterSubsystem` and `UnregisterSubsystem`, but should do that for the `GetSubsystemInfo` test.

Reply:
Updated.